client/Router/users.js

```diff
@@ -0,0 +1,20 @@
+var express = require('express');
```

Comment:
this whole file shouldn't exist - it's in client, the work you're doing is on *server*